crates/ark/src/interface.rs

```diff
@@ -337,7 +337,7 @@ impl PendingInputs {
 
         let input = if harp::get_option_bool("keep.source") {
             _srcfile = Some(SrcFile::new_virtual_empty_filename(input.into()));
-            harp::ParseInput::SrcFile(&_srcfile.unwrap())
+            harp::ParseInput::SrcFile(_srcfile.as_ref().unwrap())
```

Comment:
I slightly prefer the original version where `_srcfile`'s ownership gets moved into `input` (due to `unwrap()` taking `self`), which corresponds to the original intent.

It's fine either way, I'm mostly wondering why this unrelated change?

Reply:
I've bumped the minimum version to 1.89 on main to avoid such issues in the future.